tools/Google.Cloud.Tools.ReleaseManager/History/HistoryFile.cs

```diff
@@ -97,6 +97,8 @@ internal List<Section> MergeReleases(List<Release> releases)
 
         public sealed class Section
         {
+            private static readonly StructuredVersion ExpectedInitialReleaseVersion = StructuredVersion.FromString("1.0.0-beta01");
```

Comment:
nit: Is this the right casing/naming convention for a static field??

Reply:
Well, it's immutable, so it's sort of a constant... we go back and forth on this, to be honest.
Will change it to s_ though.